wpilibj/src/main/java/edu/wpi/first/wpilibj/smartdashboard/SmartDashboard.java

```diff
@@ -114,6 +118,9 @@ public static synchronized Sendable getData(String key) {
    * @return Network table entry.
    */
   public static NetworkTableEntry getEntry(String key) {
+    if (!m_reported) {
+      HAL.report(tResourceType.kResourceType_SmartDashboard, 0);
+    }
```

Comment:
Is this the only place we want to report?

Reply:
I don't see a reason to report when the getters are called. Does that really count as "usage" when you need to have put something there first? Teams could do it through NetworkTableInstance, but I think it's a reasonable assumption that they've called a setter at least once before calling a getter.

Reply:
I meant `getEntry()` as opposed to the other `get*` functions -- the others don't seem to have the reporting